### DISCUSSION TITLE
New version not working on Windows 7

### DISCUSSION THREAD

**Discussion by JPhB** [jgraph/drawio-desktop#1694] [Q&A]
I upgraded drawio on my Windows 7 system and encountered a problem. How do I solve it, or get a previous version that would work?

![image](https://github.com/jgraph/drawio-desktop/assets/39348213/398a6555-3276-4c20-8530-b1600f384460)

**Comment by davidjgraph** [comment]
https://github.com/jgraph/drawio-desktop/issues/1358
